.azure/app-cloud-e2e.yml

```diff
@@ -19,6 +19,7 @@ trigger:
     - "src/lightning_app/**"
     - "examples/app_*"
     - "tests/tests_app_examples/**"
+    - ".actions/assistant.py"
```

Comment:
shouldn't we rather retrigger everything when anything in `.actions` changes?

Reply:
Probably yes